backend/fms_core/services/utils.py

```diff
@@ -0,0 +1,22 @@
+import math
+
+def fit_string_with_ellipsis_in_middle(string: str, max_length: int, ellipsis = "...") -> str:
```

Comment:
Sorry about that but i would strongly advise to create a couple of tests (at least one without removal and one with) for this service as well as leave a little docstring comment for documentation sake.

Reply:
I forgive you.